webapp/src/components/Help.tsx

```diff
@@ -6,11 +6,12 @@ import table from '../images/rsu_count.PNG'
 import menu from '../images/rsu_menu.PNG'
 import heatmap from '../images/rsu_heatmap.PNG'
 import configure from '../images/rsu_configure.PNG'
+import { DotName } from '../constants'
 
 const Help = () => {
   return (
     <div id="help">
-      <h1 className="helpHeader">Welcome to the CDOT CV Manager Website</h1>
+      <h1 className="helpHeader">Welcome to the {DotName} CV Manager Website</h1>
       <div className="spacer">
         <p className="pHelp">
           This application shows the physical location and message counts for each RSU installed by the Colorado
```

Comment:
Should we add additional .env variables to pull in here so that 'Colorado Department of Transportation' and 'Colorado' are configurable?

Reply:
I think this would be better in a different PR, I will talk to Brandon about putting this on the backlog. I think there is going to be a lot of work in de-branding the cvmanager. We probably want to do something different than environment variables, maybe configuration files with a keycloak theme, images, and titles/descriptions

Reply:
Agreed, this pull request didn't create this issue and isn't addressing it so we should handle it elsewhere. Good to note though! I definitely think we need an overall customizable branding feature at some point, as Jacob suggested.